### PR TITLE
Bump JAliEn components and AliEn CA

### PR DESCRIPTION
**Pull request by Atlantic777**
- xjalienfs: the readline workaround, enable tab complete
- jalien: minor improvements in jalien shell
- AliEn CA: certificate update for OpenSSL 1.1 requirements (capabilities)

### CODE REVIEW

File: alien-cas.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package: AliEn-CAs
 version: v1
-tag: bcb18b10f695dbfe85599b7bbb5452720a91dddd
```
Comment (ktf) [inline alien-cas.sh]: Does this commit actually exist?

File: jalien.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package: JAliEn
 version: "%(tag_basename)s"
-tag: "1.1.2"
+tag: "1.1.3"
```
Comment (ktf) [inline jalien.sh]: I do not see this tag.

Comment (costing) [reply jalien.sh]: Ups! It does now

**Comment by Atlantic777** [comment]
@ktf we can proceed with this.

**Pull request merged**